assets/js/components/Button/Button.jsx

```diff
@@ -12,12 +12,16 @@ const getSizeClasses = (size) => {
 
 const getButtonClasses = (type) => {
   switch (type) {
+    case 'primary-white-fit':
```

Comment:
I created this style and `default-fit` as the current button styles use Tailwind class `w-full` which stretches the buttons to take the entire width of their container. While from following the design, I need `w-fit`, so that the width of the button is only as big as it's contents.

There may be a nicer way to achieve this. Thoughts?

Reply:
Similarly in the design docs, the `Modal` component's width is also `w-fit` instead of `w-full`, so that it is not stretched to the entire screen width. But currently the `Modal`'s `w-full` class is overriding any classes set from above. Thoughts about this @jagabomb?

![image](https://github.com/trento-project/web/assets/113615552/63703322-7781-4afe-8e90-704ea4bd1290)

![image](https://github.com/trento-project/web/assets/113615552/54474f98-2df8-4ff6-ac4d-980690e8b57e)

Reply:
> Similarly in the design docs, the `Modal` component's width is also `w-fit` instead of `w-full`, so that it is not stretched to the entire screen width. But currently the `Modal`'s `w-full` class is overriding any classes set from above. Thoughts about this @jagabomb?
Hi @jamie-suse, I do prefer the modal that is smaller in width as it wraps the content more tightly.